src/rt/cover.d

```diff
@@ -57,9 +58,35 @@ private
     __gshared
     {
         Cover[] gdata;
+        Config config;
+    }
+
+    struct Config
+    {
         string  srcpath;
         string  dstpath;
         bool    merge;
+
+    @nogc nothrow:
+
+        bool initialize()
+        {
+            return initConfigOptions(this, this.errorName);
+        }
+
+        void help()
+        {
+            string s = "Code coverage options are specified as white space separated assignments:
```

Comment:
[Whitespace](https://en.wikipedia.org/wiki/Whitespace_character)

Reply:
Do you mean I should write "whitespace" instead of "white space"? This was a copy&paste from the gc code, so if that's so that should be fixed too :)

Reply:
Updated the GC help too.